db/db.go

```diff
@@ -502,44 +503,44 @@ func (db database) GetListedPosts(r *http.Request) ([]PeopleExtra, error) {
 	return ms, result.Error
 }
 
-func (db database) GetListedWanteds(r *http.Request) ([]PeopleExtra, error) {
+func (db database) GetListedWanteds(r *http.Request) ([]Bounty, error) {
 	pubkey := chi.URLParam(r, "pubkey")
-	ms := []PeopleExtra{}
+	ms := []Bounty{}
 
-	var rawQuery string
-	var result *gorm.DB
-	// set limit
-	offset, limit, sortBy, _, search := utils.GetPaginationParams(r)
+	err := db.db.Raw(`SELECT * FROM bounty where assignee = '` + pubkey + `'`).Find(&ms).Error
 
-	if pubkey == "" {
-		rawQuery = makeExtrasListQuery("wanted")
-	} else {
-		rawQuery = makePersonExtrasListQuery("wanted")
-	}
+	return ms, err
+}
 
-	// 3/1/2022 = 1646172712, we do this to disclude early test tickets
-	rawQuery = addNewerThanTimestampToExtrasRawQuery(rawQuery, 1646172712)
```

Comment:
We can ignore for now but I'm just noting that we used to not show wanteds older than a certain date